plugins/mysql/mysql_audit

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/python
```

Comment:
Probably you meant `python3` or even `/usr/bin/env python3`?

Reply:
Indeed.